backend/typescript/services/implementations/__tests__/reviewService.test.ts

```diff
@@ -95,4 +95,59 @@ describe("pg reviewService", () => {
       expect(getReviewResult.publishedAt).toEqual(testReviews[i].publishedAt);
     });
   });
+
+  it("delete review by id", async () => {
+    // Get original reviews in DB
+    const oldResult: ReviewResponseDTO[] = await reviewService.getReviews();
+
+    const deletedId = oldResult[0].reviewId;
+    // Use the 'body' field to identify reviews for these tests, since we do
+    // not know which review was assigned to which ID in the DB
+    const deletedBodyVal = oldResult[0].body;
+
+    // Delete a review
+    await reviewService.deleteReview(deletedId.toString());
+    await expect(reviewService.getReview(deletedId.toString())).rejects.toThrow(
+      `Review id ${deletedId} not found`,
+    );
+
+    // Get updated list of reviews in DB
+    const newResult: ReviewResponseDTO[] = await reviewService.getReviews();
+    expect(newResult.length).toEqual(oldResult.length - 1);
+
+    // Sort actual and expected responses to make sure they are in the same order
+    newResult.sort((a, b) => a.body.localeCompare(b.body));
```

Comment:
What is this doing? Is it sorting the reviews based on the body?

Reply:
This is sorting the results by the 'body' field so we can ensure the ordering of the sample response matches with the actual response (and we can then check that each element in the actual response is correct). I added a comment in the sample request/response file as well to make sure that the body field is a unique number for testing purposes.

Reply:
Ahh that's why my tests were failing. Thank you!